### PR TITLE
[cob_sick_s300] adjust range_max

### PR DESCRIPTION
**Pull request by ipa-owner-jba**
replaces #334

### CODE REVIEW

**Review by ipa-rmb — APPROVED** [review]

**Comment by fmessmer** [comment]
as in #334: do not merge until tested in msh scenario

**Comment by jabbenseth** [comment]
as in #334: feel free to test in msh scenario :smile:

**Review by fmessmer — DISMISSED** [review]
we now use this feature branch on paul-ingolstadt...
we would like to get some experience in the real market before merging...please hold this for one more week or so...

**Pull request merged**
